sycl/include/CL/sycl/detail/common.hpp

```diff
@@ -166,7 +166,9 @@ static inline std::string codeToString(cl_int code) {
 #ifdef __SYCL_SUPPRESS_OCL_ERROR_REPORT
 // SYCL 1.2.1 exceptions
 #define __SYCL_CHECK_OCL_CODE(X) (void)(X)
-#define __SYCL_CHECK_OCL_CODE_THROW(X, EXC) (void)(X)
+#define __SYCL_CHECK_OCL_CODE_THROW(X, EXC, STR)                               \
+  (void)(X);                                                                   \
+  (void)(STR);
```

Comment:
I think we may get warnings without the brackets as it would expand to something like
```c++
(void)(pi_result);
(void)(message);;
```
and I vaguely recall double ; causing a warning.

Reply:
OK sure. Change made